pygmt/tests/test_which.py

```diff
@@ -18,10 +18,23 @@ def test_which():
         assert os.path.basename(cached_file) == fname
 
 
+def test_which_multiple():
+    """
+    Make sure which returns file paths for multiple @files correctly.
```

Comment:
```suggestion
    Make sure "which" returns file paths for multiple @files correctly.
```

Reply:
Good suggestion, 'which' does sound confusing. I've updated the docstrings for the other tests too (and used backticks "`" instead) in bb8206373abf452134294927696f57814947a3c7.